README.md

```diff
@@ -1,12 +1,66 @@
-# Accessible diff viewer
+---
 
-The repository contains a work in progress diff viewer with an emphasis on being accessible.
+# Diffcessible: An accessible diff viewer
 
-If you want to try it out:
+Diffcessible is an terminal based diff viewer with an emphasis on being accessible. Our goal is to build a diff viewer on top of Git diff that is accessible to screen readers.
+
+## Overview
+
+Diffcessible currently has two main components:
+
+1. `bin/main.ml`: The main entry point for the application. It is responsible for parsing the command-line arguments and calling the appropriate functions to display the diff. The key point to note here is that the command-line requires a path to a diff file as a required positional argument. See the section on [Running the Project](#running-the-Project) for more details.
+
+2. `lib/interactive_viewer.ml`: The main logic for the diff viewer. It is responsible for parsing the diff and displaying it in the terminal. This fetches the hunk information from the diff and displays it in a user-friendly manner by indexing the hunks. The core logic for the viewer is implemented in the `view` function, which allows the user to navigate through the hunks using the `n` and `p` keys to move to the next and previous hunks, respectively. The `q` key can be used to quit the viewer.
+
+## Installation
```

Comment:
This does not install the project but rather the dependencies. Perhaps it could be called "Setup Dependencies" ?

Reply:
Resolved.